CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Fixed
+- Rendering of single-image thumbnails when position and size properties are missing.
```

Comment:
This still contains technical details that someone who is simply "using" (configuring) the UI is not aware of. Also we do not write full stops :)